spark/src/test/scala/io/delta/tables/DeltaTableSuite.scala

```diff
@@ -670,6 +671,65 @@ class DeltaTableHadoopOptionsSuite extends QueryTest
     }
   }
 
+  test(
+    "dropFeatureSupport - with filesystem options.") {
```

Comment:
```suggestion
    "dropFeatureSupport - with filesystem options.") {
```
This can be on within the same line as "test"